### PR TITLE
Add Flare related networks

### PR DESCRIPTION
**Pull request by RareData**
Changes proposed in this pull request:
- Add Flare Mainnet (https://chainlist.org/chain/14)
- Add Flare Testnet Coston (https://chainlist.org/chain/16)
- Add Songbird Canary-Network (https://chainlist.org/chain/19)

### CODE REVIEW

**Comment by HariSeldon23** [comment]
Hey @ChaituVR would you be able to review when you get a chance?

File: src/networks.json [hunk 1]
```diff
@@ -122,6 +122,53 @@
...
+    "shortName": "Songbird",
+    "chainId": 19,
+    "network": "songbird",
+    "multicall": "0x17032Ea9c3a13Ed337665145364c0d2aD1108c91",
+    "rpc": [
+      "https://songbird-api.flare.network/ext/C/rpc",
+      "https://songbird.towolabs.com/rpc",
+      "https://sgb.ftso.com.au/ext/bc/C/rpc",
+      "https://sgb.lightft.so/rpc",
+      "https://sgb-rpc.ftso.eu"
```
Comment (ChaituVR) [inline src/networks.json:166]: Only the first RPC is a full archive node.
```suggestion
      "https://songbird-api.flare.network/ext/C/rpc"
```

File: src/networks.json [hunk 2]
```diff
@@ -122,6 +122,53 @@
...
+    ],
+    "explorer": "https://songbird-explorer.flare.network",
+    "start": 22501833,
```
Comment (ChaituVR) [inline src/networks.json:169]: Multicall creation block
```suggestion
    "start": 21807126,
```

File: src/networks.json [hunk 3]
```diff
@@ -122,6 +122,53 @@
...
+    "shortName": "Coston",
+    "chainId": 16,
+    "network": "coston",
+    "testnet": true,
+    "multicall": "0xF7aB82e5253F65496e21dF0dacfA6D5e765b4874",
+    "rpc": [
+      "https://coston-api.flare.network/ext/bc/C/rpc"
+    ],
+    "explorer": "https://coston-explorer.flare.network",
+    "start": 5042953,
```
Comment (ChaituVR) [inline src/networks.json:151]:
```suggestion
    "start": 4782058,
```

File: src/networks.json [hunk 4]
```diff
@@ -122,6 +122,53 @@
     "start": 657806,
     "logo": "ipfs://QmfF4kwhGL8QosUXvgq2KWCmavhKBvwD6kbhs7L4p5ZAWb"
   },
+  "14": {
+    "key": "14",
+    "name": "Flare Mainnet",
+    "shortName": "Flare",
+    "chainId": 14,
+    "network": "flare",
+    "multicall": "0x0c7B6FC9ffaBC273E9934383f258294A00Db7Fc3",
```
Comment (ChaituVR) [inline src/networks.json:131]: Hello @RareData Verify and publish contract code [here ](https://flare-explorer.flare.network/address/0x0c7B6FC9ffaBC273E9934383f258294A00Db7Fc3/contracts) plz :)

File: src/networks.json [hunk 5]
```diff
@@ -122,6 +122,53 @@
...
+    "rpc": [
+      "https://flare-api.flare.network/ext/C/rpc"
+    ],
+    "explorer": "https://flare-explorer.flare.network",
+    "start": 946850,
```
Comment (ChaituVR) [inline src/networks.json:136]:
```suggestion
    "start": 936246,
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Everything else looks good :)

**Comment by RareData** [comment]
We'll be updating the logos and some of the multicall deployments, please hold off for now.

**Comment by RareData** [comment]
> Everything else looks good :)

Ready for another review. All comments should be fixed.

**Comment by ChaituVR** [comment]
Awesome! I will take a look, Can update the base branch plz?

**Comment by RareData** [comment]
Done

**Pull request merged**